### PR TITLE
Add test showing bug in model reload with (in|out)puts info

### PR DESCRIPTION
**Pull request by atobiszei**

### CODE REVIEW

**Comment by atobiszei** [comment]
Output of undisabled test:
`[2020-10-26 09:32:40.534] [info] STATUS CHANGE: Version 0 of model dummy status change. New status: ( "state": "AVAILABLE", "error_code": "OK" )
src/test/modelinstance_test.cpp:248: Failure
Expected equality of these values:
  1
  inputsMap.size()
    Which is: 2
src/test/modelinstance_test.cpp:249: Failure
Expected equality of these values:
  1
  outputsMap.size()
    Which is: 2
b
input
a
output
[  FAILED  ] TestReloadModel.SuccessfulReloadFromAlreadyLoadedWithChangedModelMapping (59 ms)`

File: src/test/modelinstance_test.cpp [hunk 1]
```diff
@@ -231,6 +231,32 @@ TEST_F(TestReloadModel, SuccessfulReloadFromAlreadyLoaded) {
     EXPECT_EQ(ovms::ModelVersionState::AVAILABLE, modelInstance.getStatus().getState());
 }
 
+TEST_F(TestReloadModel, DISABLED_SuccessfulReloadFromAlreadyLoadedWithChangedModelMapping) {
+    std::filesystem::path dir = std::filesystem::current_path();
+    std::string dummy_model = dir.u8string() + "/src/test/dummy";
```
Comment (dkalinowski) [inline src/test/modelinstance_test.cpp:236]: Unused temp variable

**Review by dkalinowski — CHANGES_REQUESTED** [review]

**Review by dkalinowski — APPROVED** [review]

**Review by rasapala — APPROVED** [review]
LGTM

**Comment by atobiszei** [comment]
Closing since fix for that case is already merged together with this commit.

**Pull request closed without merging**